### PR TITLE
add zh translation content/zh/docs/reference/setup-tools/kubeadm/gene…

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew_apiserver.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+  </tbody>
+</table>
+
+
+
+<!-- SEE ALSO -->
+其他
+
+<!-- * [kubeadm alpha certs renew](kubeadm_alpha_certs_renew.md)	 - Renew certificates for a Kubernetes cluster -->
+* [kubeadm alpha 证书续订](kubeadm_alpha_certs_renew.md)	 - 续订 Kubernetes 集群的证书
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew_apiserver.md]: need update

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew_apiserver.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">[实验] "真实"主机根文件系统的路径。</td>
+    </tr>
+
+  </tbody>
+</table>
+
+
+
+<!-- SEE ALSO -->
+其他
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew_apiserver.md]:
```suggestion
查看其它
```

**Comment by tanjunchen** [comment]
@xichengliudui  Thanks

**Pull request merged**